project_national/resources/options_saturations.csv

```diff
@@ -0,0 +1,12381 @@
+Level,Parameter,Option,Saturation

```

Comment:
Is this file just supplemental information, from the perspective of a resstock user? Should it be rendered somehow, or at least referenced, from the docs?

Reply:
This is supplemental information. It could be rendered in the docs like how the source_report is rendered but that's optional.